lib/util/cmcd_manager.js

```diff
@@ -530,20 +530,22 @@ shaka.util.CmcdManager = class {
       this.reportEvent_('ps', {sta: 'k'}),
     );
 
-    // Fullscreen Change (Player Expand/Collapse)
-    this.eventManager_.listen(this.video_, 'fullscreenchange', () => {
+    // Fullscreen/PiP Change (Player Expand/Collapse)
+    this.eventManager_.listen(document, 'fullscreenchange', () => {
```

Comment:
With this we don't know is video/player fullscreen or some other element.

Reply:
Yeah I guess so... In the future we should consider configuring which element should be validated as a fullscreen (video/container, or some other custom one).